ddtrace/contrib/internal/requests/patch.py

```diff
@@ -59,5 +62,13 @@ def unpatch():
         return
     requests.__datadog_patch = False
 
-    _u(requests.Session, "request")
-    _u(requests.Session, "send")
+    try:
+        _u(requests.Session, "request")
+    except AttributeError:
+        # It was not patched
```

Comment:
I wonder if we should add debug logs for these cases, as some things will not work, right?

Reply:
A we don't patch everything always now, unpatch could fail trying to unpatch something not patched. So no logs required here, just normal logic. I suspect this is only used in tests anyway...